src/olympia/editors/models.py

```diff
@@ -497,8 +497,10 @@ def award_points(cls, user, addon, status, version=None,
         score = amo.REVIEWED_SCORES.get(event)
 
         # Add bonus to reviews greater than our limit to encourage fixing
-        # old reviews.
-        if version and version.nomination:
+        # old reviews. Does not apply to content-review/post-review at the
+        # moment, because it would need to be calculated differently.
+        if (version and version.nomination and
+                not post_review and not content_review):
```

Comment:
You gonna hate me and @eviljeff will probably laugh loudly… can we put that into a variable like `award_bonus` that bundles that and makes the if-statement easier to read? ❤️